docs/specs/xref.yml

```diff
@@ -211,3 +211,38 @@ outputs:
   build.manifest:
   xrefmap.json: | 
     {"references":[{"uid":"a","href":"docs/a.json","summary":"<p>Hello <code>docfx</code>!</p>"}]}
+---
+# Xref property field which supports inline markdown content
+inputs:
+  docfx.yml: |
+    buildInternalXrefMap: true
+  docs/a.json: |
+    {
+      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+      "uid": "a",
+      "inlineDescription": "Hello `docfx`!"
+    }
+outputs:
+  docs/a.json:
+  build.manifest:
+  xrefmap.json: | 
+    {"references":[{"uid":"a","href":"docs/a.json","inlineDescription":"Hello <code>docfx</code>!"}]}
+---
+# Xref property field with dependecy
+inputs:
+  docfx.yml: |
+    buildInternalXrefMap: true
+  docs/a.json: |
+    {
+      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+      "uid": "a",
+      "inlineDescription": "[!INCLUDE[B](b.md)]"
```

Comment:
>"[!INCLUDE[B](b.md)]" [](start = 27, length = 21)

the case doesn't give much differences with above "# Xref property field which supports markdown content"